docs/markdown/line-series.md

```diff
@@ -43,3 +43,13 @@ const stringCurveProp = <LineSeries data={data} curve={'curveMonotoneX'} .../>;
 const configuredCurve = d3Shape.curveCatmullRom.alpha(0.5);
 const funcCurveProp = <LineSeries data={data} curve={configuredCurve} .../>;
 ```
+
+#### style (optional)
+Type: `object`
+An object which holds CSS properties that will be applied to the SVG element(s) rendered by the series. This allows you to style series beyond the other explicitly defined properties and without having to use CSS classnames and stylesheets.
```

Comment:
Ah. You're using `series` in its plural form here (English is weird). You're right it's grammatically correct in this case to use without a definite or indefinite article, but given that you refer to `the series` right before I think it makes this plural use weird here.